scss/_variables.scss

```
@@ -1566,6 +1566,7 @@ $navbar-badge-margin-top:                   .375rem !default;
 // scss-docs-end orange-navbar-variables
 // End mod
 
+// ** Deprecated in v5.3.3
```

```suggestion
// Deprecated in v5.3.3: all `$navbar-dark-*`
```

And remove `// ** End **`

If OK for you, let's apply the same principle for the other occurrences.